tests/ui/error-codes/E0017.rs

```diff
@@ -6,8 +6,6 @@ const CR: &'static mut i32 = &mut C; //~ ERROR mutable references are not allowe
 //~| WARN taking a mutable
 
 static STATIC_REF: &'static mut i32 = &mut X; //~ ERROR E0658
```

Comment:
please test that this will still not compile if the `const_mut_refs` feature is enabled

Reply:
Yeah if I enable that feature the "cannot borrow immutable static as mutable" errors is back.